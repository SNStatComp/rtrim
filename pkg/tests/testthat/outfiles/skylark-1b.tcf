FILE outfiles/skylark.dat
TITLE  skylark-1b
NTIMES 8
NCOVARS 2
LABELS 
Habitat
Cov2
End
MISSING 999
WEIGHT Absent

comment Example 1; overdispersion only
weighting off
overdisp on
serialcor off
model 3
