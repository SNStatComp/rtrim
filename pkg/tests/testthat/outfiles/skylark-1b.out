 TRIM 3.61 :  TRend analysis and Indices for Monitoring data 
 STATISTICS NETHERLANDS 

 Date/Time: 4-7-2016 15:06:52

 Title :   skylark-1b

 Comment: Example 1; overdispersion only

 The following  5 variables have been read from file: 
 F:\TRIM\TRIM_manual_demo\skylark.dat

 1. Site             number of values:    55
 2. Time             number of values:     8
 3. Count            missing =        999
 4. HABITAT          number of values:  2
 5. COV2             number of values:  4

 Number of sites without positive counts (removed)       0

 Number of observed zero counts           0
 Number of observed positive counts     202
 Total number of observed counts        202
 Number of missing counts               238
 Total number of counts                 440

 Total count                         2536.0

 Sites containing more than 10% of the total count 
   Site Number  Observed Total    %   
         3        431.0         17.0
        37        266.0         10.5
        40        624.0         24.6

 Time Point Averages 
   TimePoint  Observations  Average   Index  
       1              25       8.52    1.00
       2              20       8.10    0.95
       3              30      10.90    1.28
       4              30      11.27    1.32
       5              28      12.75    1.50
       6              29      14.66    1.72
       7              22      17.00    2.00
       8              18      18.89    2.22

 RESULTS FOR MODEL: Effects for Each Time Point
 ----------------------------------------------


 ESTIMATION METHOD = Generalised Estimating Equations


  Total time used:  5.38 seconds

  Estimated Overdispersion     =  1.344

 GOODNESS OF FIT
  Chi-square                188.14, df    140, p 0.0041
  Likelihood Ratio          184.98, df    140, p 0.0065
  AIC (up to a constant)    -95.02

 WALD-TEST FOR SIGNIFICANCE OF DEVIATIONS FROM LINEAR TREND 
  Wald-Test             15.10, df     6, p  0.0195

 PARAMETER ESTIMATES 

 Parameters for Each Time Point

  Time    Additive    std.err.   Multiplicative  std.err.
     1      0                          1
     2     -0.3430      0.1259         0.7096      0.0894
     3     -0.1732      0.1075         0.8410      0.0904
     4     -0.1875      0.1080         0.8290      0.0896
     5     -0.0853      0.1061         0.9182      0.0974
     6      0.0213      0.1046         1.0216      0.1069
     7      0.0953      0.1071         1.1000      0.1178
     8      0.1712      0.1089         1.1867      0.1293

 Linear Trend + Deviations for Each Time 

           Additive    std.err.   Multiplicative  std.err.
 Slope      0.0485      0.0124         1.0497      0.0130

 Time Deviations
    1       0.2325      0.0625         1.2617      0.0789
    2      -0.1591      0.0777         0.8529      0.0662
    3      -0.0378      0.0627         0.9629      0.0604
    4      -0.1006      0.0625         0.9043      0.0565
    5      -0.0469      0.0585         0.9542      0.0558
    6       0.0112      0.0541         1.0113      0.0547
    7       0.0366      0.0525         1.0373      0.0544
    8       0.0640      0.0507         1.0661      0.0540

 Time INDICES
  Time     Model     std.err.     Imputed    std.err.
     1     1                       1
     2     0.7096     0.0894       0.7096     0.0894
     3     0.8410     0.0904       0.8410     0.0904
     4     0.8290     0.0896       0.8290     0.0896
     5     0.9182     0.0974       0.9182     0.0974
     6     1.0216     0.1069       1.0216     0.1069
     7     1.1000     0.1178       1.1000     0.1178
     8     1.1867     0.1293       1.1867     0.1293

 TIME TOTALS
  Time      Model    std.err.     Imputed    std.err.
     1        511         45          511         45
     2        362         35          362         35
     3        429         30          429         30
     4        423         28          423         28
     5        469         31          469         31
     6        522         32          522         32
     7        562         37          562         37
     8        606         42          606         42

 OVERALL SLOPE MODEL: Moderate increase (p<0.01) **
  Additive      std.err.   Multiplicative  std.err.
    0.0485       0.0124         1.0497      0.0130

 OVERALL SLOPE IMPUTED+(recommended): Moderate increase (p<0.01) **
  Additive      std.err.   Multiplicative  std.err.
    0.0485       0.0124         1.0497      0.0130

